flink-staging/flink-table/src/test/java/org/apache/flink/api/java/table/test/CastingITCase.java

```diff
@@ -44,29 +41,13 @@ public CastingITCase(TestExecutionMode mode){
 		super(mode);
 	}
 
-	private String resultPath;
-	private String expected = "";
-
-	@Rule
-	public TemporaryFolder tempFolder = new TemporaryFolder();
-
-	@Before
-	public void before() throws Exception{
-		resultPath = tempFolder.newFile().toURI().toString();
-	}
-
-	@After
-	public void after() throws Exception{
-		compareResultsByLinesInMemory(expected, resultPath);
-	}
-
 	@Test
 	public void testAutoCastToString() throws Exception {
 		ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
 		TableEnvironment tableEnv = new TableEnvironment();
 
 		DataSource<Tuple7<Byte, Short, Integer, Long, Float, Double, String>> input =
-				env.fromElements(new Tuple7<Byte, Short, Integer, Long, Float, Double, String>(
+				env.fromElements(new Tuple7<>(
 						(byte) 1, (short) 1, 1, 1L, 1.0f, 1.0d, "Hello"));
```

Comment:
I think merging L51 and L50 would be better. After omitting generic parameters, we don't need a newline for this.